composer.json

```diff
@@ -65,6 +65,7 @@
     "scripts": {
         "ci:php:lint": "find *.php Classes/ Configuration/ Tests/ -name '*.php' -print0 | xargs -0 -n 1 -P 4 php -l",
         "ci:php:sniff": "phpcs Classes Tests",
+        "ci:php:fix": "phpcbf Classes Tests",
```

Comment:
This should not be in the `ci` namespace as this isn't something to be run during CI, but during development. I propose `fix:php-sniff` (as there might also be another fixer using PHP_CS_Fixer later).